l10n_es_intrastat_report/migrations/14.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    cr.execute(
+        'ALTER TABLE "l10n_es_intrastat_product_declaration" RENAME "type" '
+        'TO "declaration_type"'
+    )
+
+    cr.execute(
+        """UPDATE l10n_es_intrastat_product_declaration_line
+        SET src_dest_country_code = CASE WHEN rc.code = 'GB' THEN 'XI' ELSE rc.code END
```

Comment:
But old declarations may contain real GB cases, so I think the option should be preserved as deprecated.